services/app-api/src/postgres/contractAndRates/prismaSharedContractRateHelpers.ts

```diff
@@ -97,12 +99,24 @@ function getContractReviewStatus(
             actionB.updatedAt.getTime() - actionA.updatedAt.getTime()
     )
     const latestAction = actions[0]
-    if (latestAction?.actionType === 'APPROVAL_NOTICE') {
+    if (latestAction?.actionType === 'MARK_AS_APPROVED') {
         return 'APPROVED'
     }
     return 'UNDER_REVIEW'
 }
 
+// -----
+function getConsolidatedStatus(
+    status: HealthPlanPackageStatus,
+    reviewStatus: ReviewStatus
+): ConsolidatedContractStatusType {
+    if (reviewStatus !== 'UNDER_REVIEW') {
```

Comment:
question: `ConsolidatedContractStatusType` does allow `UNDER_REVIEW`, but this function can never return `UNDER_REVIEW`. Is this because we would never have that state displayed on the dashboard? If so, I wonder if we should remove `UNDER_REVIEW` from `ConsolidatedContractStatusType`. Just a thought no changes needed, although maybe a comment explaining why we don't return `UNDER_REVIEW`.